sign/dilithium/internal/common/field_test.go

```diff
@@ -1,16 +1,34 @@
 package common
 
 import (
+	"crypto/rand"
+	"encoding/binary"
 	"flag"
-	"math/rand"
 	"testing"
 )
 
 var runVeryLongTest = flag.Bool("very-long", false, "runs very long tests")
 
+func randSliceUint32(N uint) []uint32 {
+	bytes := make([]uint8, 4*N)
+	n, err := rand.Read(bytes)
+	if err != nil {
+		panic(err)
+	} else if n < len(bytes) {
+		panic("short read from RNG")
+	}
+	x := make([]uint32, N)
+	for i := range x {
+		x[i] = binary.LittleEndian.Uint32(bytes[4*i:])
+	}
+	return x
+}
+
 func TestModQ(t *testing.T) {
-	for i := 0; i < 1000; i++ {
-		x := rand.Uint32()
+	const testTimes = 1000
```

Comment:
`testTimes` is written as `N` above. It might be a good idea to be consistent.

Reply:
N is a package parameter, not the number of tests

Reply:
Oh, gotcha.